### ISSUE TITLE
Realm package requires realm-constants.json file but does not pack it

### ISSUE THREAD

**Issue by bimusiek** [realm/realm-js#6270]
### How frequently does the bug occur?

Always

### Description

While trying to `import Realm from 'realm`;` I get an error:

```
error: Error: Unable to resolve module realm/realm-constants.json 
```

and by looking at the `package.json` of `Realm`, you can see it is not included in the `files`. 

### Stacktrace & log output

```shell
error: Error: Unable to resolve module realm/realm-constants.json from /Users/bimusiek/Developer/FreeYourMusic/app/node_modules/.pnpm/realm@12.3.0_react-native@0.71.11/node_modules/realm/dist/bundle.react-native.js:
```

### Can you reproduce the bug?

Always

### Reproduction Steps

Import Realm package in your project.

### Version

12.3.-

### What services are you using?

Atlas Device Sync

### Are you using encryption?

No

### Platform OS and version(s)

Electron

### Build environment

Which debugger for React Native: ..

**Comment by kraenhansen** [comment]
This is most likely a duplicate of #6144. I expect to merge and release [a fix for this](https://github.com/realm/realm-js/pull/6266) soon.

**Comment by kneth** [comment]
It is included in v12.3.1

**Comment by bimusiek** [comment]
@kneth  I can confirm, it is fixed in 12.3.1 as I just bumped the Realm in our app. <3 Thank you!
